storage.rb

```diff
@@ -0,0 +1,87 @@
+module Storage
+  def save_data
+    save_books
+    save_people
+    save_rentals
+  end
+
+  def save_books
+    file_path = File.join('data', 'books.json')
+    File.write(file_path, JSON.generate(books.map(&:to_h)))
+  rescue StandardError => e
+    puts "An error occurred while saving books: #{e.message}"
```

Comment:
![image](https://github.com/hasnatali1947/-OOP_school_library/assets/93039359/2c60d21b-b9d8-4bda-90b5-1464725b6903)

- [ ] It would be nice if you fix the issue raised by the previous reviewer 😨 which you can see in the image above 🤔 I see this error when I have data in the files already and then I run the console of our app and on exiting our console we see this error 😨 Kindly fix it make your app work as expected 👍


To see this error follow the below steps ⬇️

1. Remove all your JSON files
2. Run ruby main.rb
3. Create a person, book, and rental (Don't miss any of them)
4. Close your console using the 7 option
5. Now run ruby main.rb
6. now close your console again using the 7 option

You will see the error after the 6th step 🤔